runtime/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/stream/MqttKafkaSubscribeFactory.java

```diff
@@ -684,6 +707,244 @@ private int replyWindow()
         }
     }
 
+    public final class KafkaMessagesBootstrap
```

Comment:
After moving the storage of bootstrap streams from binding to subscribe factory, this class can become private.